app/components/AnalystDashboard/AssessmentAssignmentTable.tsx

```diff
@@ -338,6 +364,10 @@ const AssessmentAssignmentTable: React.FC<Props> = ({ query }) => {
         header: 'Zone',
         size: 24,
         maxSize: 24,
+        Cell: ({ cell }) => (cell.getValue() as number[]).join(', '),
+        filterVariant: 'select',
+        filterSelectOptions: uniqueZones,
+        filterFn: filterZones,
       },
       {
         ...sharedAssessmentCell,
```

Comment:
I think we're missing the PM Assessment here for the filter dropdown

Reply:
Great catch, added